ui/src/pages/loginpage/LoginWithLocalAccountForm/LoginWithLocalAccountForm.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import { getIn } from 'formik';
+import { Container } from 'semantic-ui-react';
+import isEmpty from 'lodash/isEmpty';
+import { BaseForm, StringField } from '@forms';
+import { authenticationService } from '@authentication/services';
+
+export class LoginWithLocalAccountForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: {
+        email: '',
+        password: '',
+      },
+    };
+  }
+
+  get buttons() {
+    return [
+      {
+        name: 'login',
+        content: 'Log In',
+        primary: true,
+        type: 'button',
+        fluid: true,
+      },
+    ];
+  }
+
+  onSubmit = async (values, actions) => {
+    try {
+      actions.setSubmitting(true);
+      const response = await authenticationService.loginWithLocalAccount(
+        values
+      );
+
+      this.onSubmitSuccess(response);
+    } catch (error) {
+      const errors = getIn(error, 'response.data.errors', []);
+
+      if (isEmpty(errors)) {
+        const message = getIn(error, 'response.data.message', null);
```

Comment:
`getIn` `error`? Is it correct?

Reply:
Yes as errors are empty in that case so it falls back to the `response.data.message` :)